example/lib/src/app.dart

```diff
@@ -100,8 +100,7 @@ class _ExampleState extends State<Example> {
   Future<void> _initializeAppcues() async {
     AppcuesOptions options = AppcuesOptions();
     options.logging = true;
-    await Appcues.initialize(
-        'APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID', options);
+    await Appcues.initialize('APPCUES_ACCOUNT_ID', 'APPCUES_APPLICATION_ID', options);
```

Comment:
doublechecking - did this pass flutter format? I think `dart format` does it. Otherwise it will knock points of our pub score https://pub.dev/help/scoring

Reply:
i WIll doublecheck, thanks